package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.2.2",
   "author": "Izaak Schroeder <izaak.schroeder@gmail.com>",
   "scripts": {
-    "prepublish": "./node_modules/.bin/babel -d ./ ./src/",
+    "prepublish": "./node_modules/.bin/babel  -s -d ./ ./src/",
```

Comment:
Theoretically no. Practically it catches people who have `babel` installed globally. I'll probably change this once CI handles publishing.